mail/mail-server/Portfile

```diff
@@ -329,6 +341,29 @@ destroot {
 
 destroot.keepdirs ${destroot}${prefix}/var/log/mail
 
+# Workaround for issue with dovecot version 2.3.17 on macOS 12
+# https://www.mail-archive.com/dovecot@dovecot.org/msg84784.html
+if { ${os.major} >= 21 } {
+    post-destroot {
+        # default_vsz_limit = 0 (append after last appearance)
+        reinplace -E "/^\[\[:space:]]*#\[\[:space:]]*default_vsz_limit\[\[:space:]]+=\[\[:space:]]+\[^#\]*/,$!b
+//{x;//p;g;};//!H;$!d;x;s//&# Workaround for issue with dovecot version 2.3.17 on macOS 12\\
+# https:\\/\\/www.mail-archive.com\\/dovecot@dovecot.org\\/msg84784.html\\
+default_vsz_limit = 0\\
+\\
+/" \
+            ${destroot}${prefix}/etc/dovecot/conf.d.macports/10-master.conf
+
+        # service imap-login vsz_limit = 0
+        reinplace -E "s/^(\[\[:space:]]*)(vsz_limit\[\[:space:]]+=\[\[:space:]]+\[^#\]*)/\\1# \\2\\
+\\
+\\1# Workaround for issue with dovecot version 2.3.17 on macOS 12\\
+\\1# https:\\/\\/www.mail-archive.com\\/dovecot@dovecot.org\\/msg84784.html\\
+\\1vsz_limit = 0/g" \
+            ${destroot}${prefix}/etc/dovecot/conf.d.macports/10-master.conf
+    }
```

Comment:
these should be patchfiles and not these unreadable reinplaces

Reply:
Thanks, done.